src/main/java/stepsdesign/PipelineSteps.java

```diff
@@ -659,4 +663,58 @@ public void clickOnCreateButton () {
   public void verifyInvalidConnectionErrorMessage(String errorMessageLocation) {
     CdfConnectionActions.verifyConnectionErrorMessage(errorMessageLocation);
   }
+
+  @Given("Open DataFusion Project with replication to configure pipeline")
+  public void openDataFusionProjectWithReplicationToConfigurePipeline() throws IOException, InterruptedException {
+    openCdf();
+    SeleniumDriver.getDriver().get(SeleniumDriver.getDriver().getCurrentUrl().replace(
```

Comment:
Same here, create a method in ```CdfPluginPropertiesActions``` and use it.

Reply:
Done.